Source/Mosa.Kernel.x86/Panic.cs

```diff
@@ -216,9 +236,23 @@ public static void Error(string message)
 			Halt();
 		}
 
+		// TODO: Try to reduce redundancy. This is not simple, because delegates ans dynamic strings are not allowed.
+		// Onother way idea is to increate the size of the StringBuffer and call Error(new StringBuffer(message))
```

Comment:
// TODO: Try to reduce redundancy. This is not simple, because delegates and runtime strings cannot be used.
// Another way is to increment the size of the StringBuffer and call Error(new StringBuffer(message))